docs/Ecosystem.md

```diff
@@ -13,6 +13,8 @@ Plugins maintained by the fastify team are listed under [Core](#core) while plug
 - [`fastify-basic-auth`](https://github.com/fastify/fastify-basic-auth) Basic auth plugin for Fastify.
 - [`fastify-bearer-auth`](https://github.com/fastify/fastify-bearer-auth) Bearer auth plugin for Fastify.
 - [`fastify-caching`](https://github.com/fastify/fastify-caching) General server-side cache and etag support.
+- [`fastify-casbin`](https://github.com/nearform/fastify-casbin) Casbin support for Fastify.
+- [`fastify-casbin-rest`](https://github.com/nearform/fastify-casbin-rest) Casbin support for Fastify based on a RESTful model.
```

Comment:
Those are Ecosystem plugins, not core plugins, see https://github.com/fastify/fastify/pull/2675/files#diff-6e498fbb74d36e22505f94fa2f083a9f3f3741aae2534c96e860d0daffb516e9R51

Reply:
ahh my bad